setup.py

```diff
@@ -29,7 +29,8 @@
         'nbconvert>=5.3,!=5.4',
         'traitlets>=5',
         'nbformat',
-        'sphinx>=1.8',
+        # https://github.com/spatialaudio/nbsphinx/issues/825
+        'sphinx >= 1.8, < 8.2',
```

Comment:
See my comment in https://github.com/spatialaudio/nbsphinx/issues/825#issuecomment-2692944098, I think that Sphinx 8.2.2 shoud now work with `nbsphinx` (though the explicit imports in #826 remain the correct solution)
```suggestion
        'sphinx >= 1.8, != 8.2.0, != 8.2.1',
```

Reply:
@AA-Turner @mgeier I can confirm that sphinx 8.2.2 passes with current nbsphinx for now

Reply:
Thanks for the update!

I'm still getting a warning related to https://github.com/felix-hilden/sphinx-codeautolink/issues/173.

I will un-pin Sphinx when this is done. I might also re-think the testing strategy for warnings, because keeping up with them is quite time-consuming.

I have just made a new `nbsphinx` release which still blocks Sphinx 8.2+.

Reply:
Indeed, to Matthias' point about testing strategy for warnings, enabling deprecation warnings for the documentation build (or non-core dependencies) probably isn't needed and may be unhelpful in this case.